### PR TITLE
admin show users list and destroy user function

### PR DESCRIPTION
**Pull request by VuongLong**

### CODE REVIEW

File: app/controllers/admin/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class Admin::UsersController < ApplicationController
+  load_and_authorize_resource
+
+  def index
+    @users = User.page(params[:page]).per(5)
```
Comment (hellovietnam93) [inline app/controllers/admin/users_controller.rb]: remove ()

**Comment by hellovietnam93** [comment]
OK

**Comment by ttuan** [comment]
OK

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -1,4 +1,11 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
   before_action :authenticate_user!
+
+  include ApplicationHelper
+  
+  rescue_from CanCan::AccessDenied do |exception|
+    flash[:alert] = exception.message
+    redirect_to current_user.supervisor? ? root_path : (current_user.admin? ? root_path : new_user_session_path)
```
Comment (dieunb) [inline app/controllers/application_controller.rb]: refactor nhé

**Pull request merged**